api/controllers/posts.js

```diff
@@ -22,6 +22,23 @@ const PostsController = {
       res.status(201).json({ message: 'OK', token: token });
     });
   },
-};
+  Update: (req, res) => {
+    const updated_post = req.body.message;
+    let post_id = req.params.id;
```

Comment:
I would suggest to always prefer using `const` instead of `let` - the difference is that values declared with `let` can be changed later. However we probably don't want to let this happen here — the post ID provided in the URL parameters should stay the same in this controller function. Otherwise, someone could potentially add code like this a few lines below...

```js
post_id = 123
```

and would break the code!

Reply:
This is now changed. The pull request will be updated, once we have implemented the authorization of the user editing the post.